### PR TITLE
Moved Strings in toast messages to strings.xml

### LINKED ISSUE #223: Strings should be stored in resources(R.strings) and typo in Toast message
The strings for the messages in the toasts are hard-coded but they should be stored in the `strings.xml` file. I would like to move a lot of these strings in toast messages and other locations to `strings.xml` . This would make the code cleaner.
It would be on similar lines to #47 
Also there is a typo in one of the toast messages:
Here it should be `"Reminder time missing"`

g_20170126_1320153

@anubhakushwaha @chhavip Can I send a quick PR for this one?

### PR DESCRIPTION
**Pull request by nitish1211**
Fixes #223

This pull request moves all the hardcoded strings in the toast messages to `strings.xml` .

@chhavip @anubhakushwaha  Please review this and let me know if you think I should make any changes.

### CODE REVIEW

File: malaria-app-android/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -129,4 +129,39 @@
...
+    <string name="arr_date_before_dept">Arrival date cannot be before the Departure date.</string>
+    <!-- DayFragmentActivity -->
+    <string name="no_edit">You are not allowed to edit!</string>
+    <string name="took_medicine">I took medicine!</string>
+    <string name="no_medicine">I didn't take medicine!</string>
+    <string name="missed_entering_medicine">I missed entering medicine!</string>
+    <string name="future_cannot_edit">This is a future date, you cannot edit it!</string>
+    <string name="missed_entering">I missed entering medicine!</string>
+    <!-- EffectivenessFragmentActivity -->
+    <string name="error_loading_cache">Error Retreiving Data! Loading from cache... </string>
```
Comment (yatna) [inline malaria-app-android/src/main/res/values/strings.xml]: Fix typo . I think it should be 'Retrieving'

**Comment by nitish1211** [comment]
@yatna Done

**Comment by m-murad** [comment]
@MeepyMay 
Can you please tell which is the active branch of this repo?
I see some commiters merging PR to `develop` branch and some to `master` branch.
Can you please confirm this?

**Comment by m-murad** [comment]
@geekanamika 
before approving please check if the PR is passing the Travis check.

**Comment by geekanamika** [comment]
No update on squashing the commits, no updates travis build failed, closing the PR.

**Pull request closed without merging**